### ISSUE TITLE
"Discarding chunk message" every 3 seconds

### ISSUE THREAD

**Issue by Sheppowed** [goatchurchprime/two-voip-godot-4#39]
Hi, I implemented the addon in my project using the minimal implementation method, it works very well but the ‘Discarding chunk’ message is sent every 3 seconds and it's annoying. What does it mean and how can i hide/fix that ?

**Comment by goatchurchprime** [comment]
I agree that message is quite annoying (it used to fire 50 times a second and in the editor, so I toned it down).  

I think it's trying to tell you that the opus buffer is spinning away with input from the microphone and it's not being collected. Either set the microphone to be not playing, or disable the Effect and it should stop collecting the audio data.  

There needs to be some kind of warning to encourage you not to leave it on unnecessarily consuming CPU cycles.  I should probably make it emit one red warning error about it after 15 seconds or so of starting up with the above advice.

**Comment by ajlennon** [comment]
Why don't you check the CPU cycles and issue a warning if it hits a trigger level
